pkg/cmd/admin/groups/new.go

```diff
@@ -75,7 +82,7 @@ func (o *NewGroupOptions) Complete(f *clientcmd.Factory, args []string) error {
 	return nil
 }
 
-func (o *NewGroupOptions) AddGroup() error {
+func (o *NewGroupOptions) AddGroup(mapper meta.RESTMapper, shortOutput bool) error {
```

Comment:
If this method might produce output, then force the Options to include sufficient information to perform that output.